packages/snap-preact-components/src/components/Atoms/LoadMore/readme.md

```diff
@@ -0,0 +1,123 @@
+# LoadMore
+
+Renders a "Load More" button that displays the next page of results. Recommended to be used with SearchController's `infinite` config
+
+## Sub-components
+- Button
+
+## Usage
+
+### pagination / controller
+The required `pagination` (or `controller`) prop specifies a reference to the pagination store object.
+
+```jsx
+<LoadMore pagination={controller.store.pagination} />
+```
+
+```jsx
+<LoadMore controller={controller} />
+```
+
+### autoFetch
+The `autoFetch` prop will not render a "Show More" button and instead fetch the next page of results when the component appears in the viewport. Recommended to place underneath Results to allow for infinite scrolling. 
+
+```jsx
+<LoadMore pagination={controller.store.pagination} autoFetch={true} />
+```
+
+Can be used in combination with `` and `` to not display anything and have infinite scrolling still function:
+
+```jsx
+<LoadMore pagination={controller.store.pagination} autoFetch={true} hideProgressIndicator={true} hideProgressText={true} />
+```
+
+### intersectionOffset + autoFetch
+Requires `autoFetch` prop to be `true`. The `intersectionOffset` prop defines the IntersectionObserver's `rootMargin` value. This is an offset added to each side of the component's bounding box to create the final root bounds. These bounds then determine when the component is in the viewport and the next results are fetched. Default: `0px`
+
+```jsx
+<LoadMore pagination={controller.store.pagination} autoFetch={true} intersectionOffset={'40px'} />
+```
+
+```jsx
+<LoadMore pagination={controller.store.pagination} autoFetch={true} intersectionOffset={'100%'} />
+```
+
+### loading
+The `loading` prop is not required if the `controller` prop is provided or `hideProgressIndicator` and `hideProgressText` are both `true`
```

Comment:
Looking at the code, it doesn't seem that `hideProgressIndicator` and `hideProgressText` have anything to do with the loading spinner.

Reply:
Good catch - removed this text.